src/group/group.controller.ts

```diff
@@ -2,54 +2,82 @@ import config from "../config";
 import { Request, Response } from "express";
 import { GroupManager } from "./group.manager";
 import { IGroup } from "./group.interface";
-import { GroupNotFound, IdInvalidError } from "../utils/erros/userErrors";
+import { GroupNotFound, IdInvalidError, UnitNotFound } from "../utils/erros/userErrors";
 import { Validations } from "../utils/validations/validations";
 import { ServerError } from "../utils/erros/errorTypes";
+import { IResponseUnitSums, IResponseGetByMany } from "../response/response.interface";
+
+const groupNotFoundMsg = "Group not found";
```

Comment:
Why use this constant and not the regular constructor with groupID as the param?

Reply:
nice comment. done